src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPageNew.tsx

```diff
@@ -0,0 +1,223 @@
+import Pagination from '@common/components/Pagination';
+import RelatedInformation from '@common/components/RelatedInformation';
+import { PublicationSummaryWithRelease } from '@common/services/publicationService';
+import Link from '@frontend/components/Link';
+import Page from '@frontend/components/Page';
+import PublicationSummary from '@frontend/modules/find-statistics/components/PublicationSummary';
+import { testPublications } from '@frontend/modules/find-statistics/__tests__/__data__/testPublications';
+import { NextPage } from 'next';
+import React, { useEffect, useRef, useState } from 'react';
+import { useRouter } from 'next/router';
+
+// TODO EES-3517 some fake paging data
+const tempPaging = {
+  page: 1,
+  pageSize: 10,
+  totalResults: 100,
+  totalPages: 10,
+};
+
+interface Props {
+  paging?: {
+    // won't be optional
+    page: number;
+    pageSize: number;
+    totalResults: number;
+    totalPages: number;
+  };
+  publications?: PublicationSummaryWithRelease[]; // won't be optional
+}
+
+const FindStatisticsPageNew: NextPage<Props> = ({
+  paging = tempPaging,
+  publications = testPublications,
+}) => {
+  const router = useRouter();
+  const resultsRef = useRef<HTMLDivElement>(null);
+
+  // TODO EES-3517 - update totalResults when search / filter
+  const { totalResults } = paging;
+  // const [totalResults, setTotalResults] = useState<number>(paging.totalResults);
+  const [currentPage, setCurrentPage] = useState<number>(paging.page);
+  const [currentPublications, setCurrentPublications] = useState<
+    PublicationSummaryWithRelease[]
+  >(publications);
+
+  useEffect(() => {
+    // TODO EES-3517 - fetch selected page of publications here,
+    // something like this:
+    // async function fetchPublications(params) {
+    //   const nextPublications = await publicationService.getPublications(params);
+    //   setCurrentPublications(nextPublications);
+    // }
```

Comment:
Would suggest that we do all data loading via `getServerSideProps` to keep it all in one place. This should make things less complicated as we only need to work with the props that are passed through from there. We shouldn't need to maintain current state for page/publications with this approach.

Importantly, we would still need to handle loading states when clicking through pages, so we'll need something like this:

```ts
const router = useRouter();
const [isLoading, toggleLoading] = useToggle(false);

useEffect(() => {
  router.events.on('routeChangeStart', toggleLoading.on);
  router.events.on('routeChangeComplete', toggleLoading.off);

  return () => {
    router.events.off('routeChangeStart', toggleLoading.on);
    router.events.off('routeChangeComplete', toggleLoading.off);
  };
}, [toggleLoading, router]);
```

Would also be tempted to wrap this up into a re-usable frontend hook called `useRouterLoading`!